### PR TITLE
New docker image - OpenCTI-v4

### PR DESCRIPTION
**Pull request by daryakoval**
## Status
Ready

## Description
New docker image - OpenCTI-v4

### CODE REVIEW

File: .dependabot/config.yml [hunk 1]
```diff
@@ -1953,42 +1953,43 @@ update_configs:
     - match:
         dependency_name: demisto/*
         update_type: all
-        
+
   - package_manager: python
-    directory: /docker/opencti-v4.1.2
+    directory: /docker/xgboost
```
Comment (glicht) [inline .dependabot/config.yml:1958]: @daryakoval something is weird with the changes you are putting in the dependabot config file. Can you please check.

**Comment by glicht** [comment]
@darkushin @ShahafBenYakir we have the following image: https://github.com/demisto/dockerfiles/tree/master/docker/opencti

Why do we need this one?

**Comment by daryakoval** [comment]
@glicht - this https://github.com/demisto/dockerfiles/tree/master/docker/opencti working with OpenCTI integration, 
and the new docker works with OPENCTI v2. 
we cant use the same docker since the API changed and it breaks backward compatibility.

File: .dependabot/config.yml (hunk 1 above)
Comment (daryakoval) [reply .dependabot/config.yml:1957]: git looks at this in strange way.. I just deleted one docker and added another..

Comment (daryakoval) [reply .dependabot/config.yml:1957]: Instead of deleting previous docker image and add new one... it shifts all docker names down

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**